base/linalg/generic.jl

```diff
@@ -957,7 +957,7 @@ true
 function istriu(A::AbstractMatrix)
     m, n = size(A)
     for j = 1:min(n,m-1), i = j+1:m
-        if A[i,j] != 0
+        if A[i,j] != zero(eltype(A))
```

Comment:
I didn't know about `iszero` - it doesn't seem to be in the documentation.
But this is good.  I will update it to use `iszero`.
Thanks.